From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Tue, 7 Jan 2014 21:46:18 +0100
Subject: Don't run tests that require qemu:///system

We only want to run tests for now that work with the testdriver
---
 Rakefile | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/Rakefile b/Rakefile
index 4155f0e..015308a 100644
--- a/Rakefile
+++ b/Rakefile
@@ -75,11 +75,9 @@ task :build => LIBVIRT_MODULE
 #
 
 Rake::TestTask.new(:test) do |t|
-    t.test_files = [ 'tests/test_conn.rb', 'tests/test_domain.rb',
-                     'tests/test_interface.rb', 'tests/test_network.rb',
-                     'tests/test_nodedevice.rb', 'tests/test_nwfilter.rb',
-                     'tests/test_open.rb', 'tests/test_secret.rb',
-                     'tests/test_storage.rb', 'tests/test_stream.rb' ]
+    t.test_files = [ 'tests/test_nodedevice.rb',
+                     'tests/test_open.rb',
+                     'tests/test_stream.rb' ]
     t.libs = [ 'lib', 'ext/libvirt' ]
 end
 task :test => :build
